samples/mqtt_example/cloudiot_mqtt_example_nodejs.js

```diff
@@ -54,6 +52,15 @@ const createJwt = (projectId, privateKeyFile, algorithm) => {
 };
 // [END iot_mqtt_jwt]
 
+//iot-sandbox.clearblade.com
+const hostFormatted = hostRegion => {
```

Comment:
After trying this out, I think we should remove the hostFormatted function. That way we can the sample can be used for instances other than the production instance. Here's the command I used to connect to our community instance:
```
node cloudiot_mqtt_example_nodejs.js mqttDeviceDemo --projectId=iot-core-test-363819 --cloudRegion=us-central1 --registryId=testmonitoring --deviceId=one --serverCertFile=/Users/clarkbynum/Downloads/community.clearblade.com.cer --algorithm=RS256 --privateKeyFile=/Users/clarkbynum/Desktop/device-credentials/one/rsa_private.pem --messageType=state --mqttBridgeHostname=community.clearblade.com --mqttBridgePort=1884
```

Reply:
Yes, there is already a way to pass hots name into the parameter of 'mqttBridgeHostname' but in that case, I think we need to send all documents to the customer or read me file and they have to put broker based on the region. So, what's the best approach?

Reply:
I think requiring mqttBridgeHostname is the best approach as it's the most flexible. We already have docs for the mqtt URLs here - https://clearblade.atlassian.net/wiki/spaces/IC/pages/2210299905/Re-targetting+Devices#Production-URL-%2F-URLs